terraform/environments/analytical-platform-compute/helm-charts-actions-runners.tf

```diff
@@ -132,7 +132,7 @@ resource "helm_release" "actions_runner_mojas_create_a_derived_table_emds_test"
         github_organisation  = "moj-analytical-services"
         github_repository    = "create-a-derived-table"
         github_runner_labels = "electronic-monitoring-data-test"
-        eks_role_arn         = "arn:aws:iam::396913731313:role/emds-test-data-api-cross-account-role"
```

Comment:
Hiya, any chance you can change the hard coded account number to a local variable? I believe this is an MP account?

Reply:
yep :)